modules/apps/27-interchain-accounts/controller/keeper/msg_server_test.go

```diff
@@ -18,7 +16,7 @@ import (
 
 func (suite *KeeperTestSuite) TestRegisterAccount() {
 	var (
-		msg               *controllertypes.MsgRegisterAccount
+		msg               *types.MsgRegisterAccount
```

Comment:
if we have `icatypes`, `clienttypes` `channeltypes` etc. does it make sense to have a package imported as just `types`? It's not immediately obvious which sort of types it is, I would lean in favour of keeping it called `controllertypes` and removing the unaliased one WDYT?

Reply:
In a lot of other packages we normally import the relative `types` pkg as is, and then for any `types` pkgs from other modules we alias them. 

I don't really feel strongly about it here. It's also a test file so.. 🤷

Reply:
For interchain accounts it may help to always alias tho. Considering there is the shared `icatypes` and then a separate `types` for both controller and host submodules

Reply:
ah I see okay, so we generally just do external types with the alias, that seems fine with me 👍